libs/cardano-ledger-core/test/Test/Cardano/Ledger/AddressSpec.hs

```diff
@@ -56,6 +57,13 @@ spec = do
     prop "RewardAcnt (fromCborRewardAcnt)" $
       roundTripExpectation @(RewardAcnt StandardCrypto) (mkTrip toCBOR fromCborRewardAcnt)
 
+propSameAsOldDecompactAddr :: forall c. Crypto c => CompactAddr c -> Property
+propSameAsOldDecompactAddr cAddr =
+  (addr === decompactAddrOld @c cAddr)
+    .&&. (addr == decompactAddrOldLazy @c cAddr)
+  where
+    addr = decompactAddr @c cAddr
+
```

Comment:
Curious, if we wrote this as an Expectation instead: 
```
  do
    addr `shouldBe` decompactAddrOld @c cAddr
    addr `shouldBe` decompactAddrOldLazy @c cAddr
  where
    addr = decompactAddr @c cAddr
```

Then would we get finer grained information about whether the first or the second  line failed?  That's what I remember from the meeting, at least :thinking:

Reply:
You are absolutely right, I'll change the implementation to your suggestion

Reply:
Done